templates/SSO/aws-sso.yaml

```diff
@@ -25,6 +25,11 @@ Parameters:
     Default: ''
     Description: 'A list of AWS managed policy ARNs for a SSO permission set'
 
+  customerManagedPolicyName:
+    Type: String
```

Comment:
can we make this a list, similar to `managedPolicies`, so that users can pass in a list of customer policies?

Reply:
I think that would require a macro to parse the list of strings and turn it into a list of objects expected by the parameter, similar to my cost categories rules generator macro, and I didn't want to put in that much effort when we only have a need for a single customer managed policy.

Reply:
ahh, it's because customer managed policies would be a dictionary of items.  i think the solution is to templatetize it using org-formation's [TemplatingContext](https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#templating).  Fist step is to convert this template to a nunjucks template. Then pass in a dictionary value for `customerManagedPolicyNames` with  TemplatingContext then reference the dict in the template.

Reply:
oh, awesome! thanks

Reply:
I've converted the template to nunjucks.